qemu/tests/cfg/nested_vsock_con_sockets.cfg

```diff
@@ -10,8 +10,7 @@
     vsocks = vhost_vsock0
     vsock_port = 2345
     no RHEL.7, RHEL.8.0, RHEL.8.1, RHEL.8.2
-    RHEL.8:
-      vsock_test_tool = nc_vsock
-    RHEL.9:
-      vsock_test_tool = ncat
+    vsock_test_tool = nc_vsock
+    ! RHEL.8:
+        vsock_test_tool = ncat
```

Comment:
Suggest:
```
vsock_test_tool = ncat
RHEL.8:
    vsock_test_tool = nc_vsock
```

Means the default one is `ncat`, for rhel8 specific is `nc_vsock`.

Reply:
update.